packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -0,0 +1,302 @@
+<script lang="ts">
+import {
+  faArrowUpRightFromSquare,
+  faEllipsisVertical,
+  faPlay,
+  faRotateRight,
+  faStop,
+  faTrash,
+} from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { providerInfos } from '../../stores/providers';
+import type { ProviderContainerConnectionInfo, ProviderInfo } from '../../../../main/src/plugin/api/provider-info';
+import { onMount } from 'svelte';
+import type {
+  IConfigurationPropertyRecordedSchema,
+  IProviderContainerConfigurationPropertyRecordedSchema,
+} from '../../../../main/src/plugin/configuration-registry';
+import { configurationProperties } from '../../stores/configurationProperties';
+import type { ContainerProviderConnection } from '@tmpwip/extension-api';
+
+interface containerStatus {
+  status: string;
+  inProgress: boolean;
+}
+
+export let properties: IConfigurationPropertyRecordedSchema[] = [];
+let providers: ProviderInfo[] = [];
+$: containerConnectionStatus = new Map<string, containerStatus>();
+
+let configurationKeys: IConfigurationPropertyRecordedSchema[];
+$: containerConnectionDivWidth = 0;
+$: containerConnectionDivHeight = 0;
+
+onMount(() => {
+  configurationProperties.subscribe(value => {
+    properties = value;
+  });
+
+  providerInfos.subscribe(providerInfosValue => {
+    providers = providerInfosValue;
+    providers.forEach(provider => {
+      provider.containerConnections.forEach(container => {
```

Comment:
what happens if a provider has been removed ?

Reply:
The provider will not be displayed anymore as the providers array has been updated and the page gets refreshed, no?

Reply:
My question was more: should we display EmptyScreen/stuff to tell the user to do some actions in order to have resources